widget/entry.go

```diff
@@ -59,12 +59,17 @@ type Entry struct {
 
 	cursorAnim *entryCursorAnimation
 
+	dirty       bool
 	focused     bool
 	text        *textProvider
 	placeholder *textProvider
 	content     *entryContent
 	scroll      *widget.Scroll
 
+	// useful for Form validation (as the error text should only be shown when
```

Comment:
This is a change in behaviour - before now we would show the details of an error if there was any.
We hid the error status if the user had not yet edited the field.

Is this the desired change?

Reply:
> now we would show the details of an error if there was any.

Not sure what you mean here, that is already the entry behavior.

> We hid the error status if the user had not yet edited the field.

The only behavior change is that now Entry is going to show the error status (if there is any error) after the user has visited the input by the first time. In that way we can ensure that "not empty" validators are shown as soon as the user focused the input and passed it without solving the error.